CMakeLists.txt

```diff
@@ -1,9 +1,22 @@
 esma_set_this ()
 
-esma_add_library (${this}
-  SRCS GEOS_OceanGridComp.F90
-  SUBCOMPONENTS MOM6_GEOSPlug MOM_GEOS5PlugMod GEOSdatasea_GridComp
-  DEPENDENCIES GEOSdatasea_GridComp MAPL esmf)
+#target_include_directories (${this} PUBLIC ${INC_ESMF} ${INC_NETCDF})
+
+option (BUILD_MIT_OCEAN "BUILD_MIT_OCEAN is turned off by default" OFF) #OFF by default
+if ( NOT BUILD_MIT_OCEAN )
+  esma_add_library (${this}
+    SRCS GEOS_OceanGridComp.F90
+    SUBCOMPONENTS MOM6_GEOSPlug MOM_GEOS5PlugMod GEOSdatasea_GridComp
+    DEPENDENCIES GEOSdatasea_GridComp MAPL esmf)
+
+else()
+  add_compile_definitions(BUILD_MIT_OCEAN)
```

Comment:
This essentially means:
- When you choose to `BUILD_MIT_OCEAN`,
- You get **only** `MIT_OCEAN`.

Is that what you meant?

Reply:
yes, that was the idea. if we are incurring the memory burden of mit then we are not running mom.

Reply:
Just wanted to make sure of that (=not build). Of course, no build ➡️ can't run!